stable/aws-load-balancer-controller/README.md

```diff
@@ -71,12 +71,6 @@ Add the EKS repository to Helm:
 helm repo add eks https://aws.github.io/eks-charts
 ```
 
-Install the TargetGroupBinding CRDs:
```

Comment:
If the chart is already installed, helm upgrade will not apply the CRDs. This will prevent the users from getting the recent CRD changes required by the new chart release.

We can instead indicate that this step is optional for fresh chart install.